core-bundle/src/Composer/ScriptHandler.php

```diff
@@ -45,7 +45,7 @@ public static function generateRandomSecret(Event $event): void
             self::loadRandomCompat($event);
         }
 
-        putenv(static::RANDOM_SECRET_NAME.'='.bin2hex(random_bytes(32)));
+        putenv(self::RANDOM_SECRET_NAME.'='.bin2hex(random_bytes(32)));
```

Comment:
I disagree with this change. The method is `public`, not `private`. A child class of the `ScriptHandler` can very well override the class constant, so this method should reflect that.

Reply:
Reverted in 81c51142b035be604674720dc232958b41f0149c.